app/controllers/matchups_controller.rb

```diff
@@ -67,25 +67,16 @@ def update
   def index
     @this_tournament = Tournament.find(params[:tournament_id])
     # This order keeps the matchups in order when a score is submitted.
-    @all_matchups = @this_tournament.matchups.order(:round_number, :id)
-    @all_matchups = @all_matchups.reject { |matchup| matchup.player1_score == -50 || matchup.player2_score == -50 }
+    @all_matchups = @this_tournament.matchups.active.order(:round_number, :id)
```

Comment:
I think it would be good to ultimately put this in an if statement - 
`if @this_tournament.user == current_user` then @all_matchups is lines 70/71 (directors should be able to see/edit all matchups, even inactive ones), then `else` for the line you have

Reply:
agreed. I feel like it can be done later because there is a lot of the app that will behave differently for directors, regular logged in players, and someone who hasn't logged in but just wants to see the tournament.